src/utils.ts

```diff
@@ -26,7 +27,8 @@ export const fetchPulls = async (
   const {data: allPulls} = await octokit.rest.pulls.list({
     owner,
     repo,
-    per_page: 100
+    per_page: 100,
+    direction: 'asc'
```

Comment:
Can you add the `sort` parameter too (even though it's the default setting)? And maybe a short comment like "prioritize old PRs"?

It would make it more obvious what's going on here.